users/keppenj/index.md

```diff
@@ -4,4 +4,12 @@ layout: default
 author: Jackson Keppen
```

Comment:
The top of your index.md needs to look like this screencap, in order to show up. Just replace my name with yours.
<img width="338" alt="screenshot 2017-01-25 21 31 21" src="https://cloud.githubusercontent.com/assets/6990710/22320960/aa6f2316-e345-11e6-933d-e28d62dd5b99.png">

The syntax is very specific for the index.md files